src/test/java/org/opensearch/ad/transport/DeleteAnomalyResultsTransportActionTests.java

```diff
@@ -35,4 +49,4 @@ public void testDeleteADResultAction() throws IOException, InterruptedException
         }, 90, TimeUnit.SECONDS);
         assertEquals(1, deleteADResultResponse.getDeleted());
     }
-}*/
+}
```

Comment:
Note to other reviewers: THIS is why you don't use `//` for mass code comments. What a beautiful (lack of) diff!